Cargo.toml

```diff
@@ -53,6 +53,10 @@ walkdir = "2"
 webpki = "0.22"
 x509-parser = "0.14"
 zeroize = "1.5"
+lazy_static = "1.4.0"
```

Comment:
please use https://docs.rs/once-cell instead, it has a lower compile time & dependency overhead

Reply:
And is in std on nightly IIUC!  Or at least a subset.